notebooks/00-getting_started.ipynb

```diff
@@ -194,7 +172,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prediction = axon_segmentation([path_testing], [\"image.png\"], path_model, config_network, resampled_resolutions=0.1, verbosity_level=3)"
+    "segment_image(\n",
```

Comment:
At line 164 above, the names of the model should be updated.
At line 166 above, the name of the output file should be updated.

Reply:
Resolved in 2dbc5f768